swatch-contracts/src/main/resources/application.properties

```diff
@@ -319,12 +324,26 @@ mp.messaging.outgoing.capacity-reconcile.topic=platform.rhsm-subscriptions.capac
 mp.messaging.outgoing.capacity-reconcile.group.id=capacity-reconciliation-worker
 mp.messaging.outgoing.capacity-reconcile.value.serializer=io.quarkus.kafka.client.serialization.ObjectMapperSerializer
 
-mp.messaging.incoming.offering-sync-umb.connector=smallrye-amqp
-%test.mp.messaging.incoming.offering-sync-umb.connector=smallrye-in-memory
-mp.messaging.incoming.offering-sync-umb.address=${OFFERING_UMB_QUEUE}
-mp.messaging.incoming.offering-sync-umb.client-options-name=umb
-mp.messaging.incoming.offering-sync-umb.enabled=${UMB_ENABLED}
-mp.messaging.incoming.offering-sync-umb.failure-strategy=accept
+mp.messaging.incoming.offering-sync-canonical-umb.connector=smallrye-amqp
+%test.mp.messaging.incoming.offering-sync-canonical-umb.connector=smallrye-in-memory
+mp.messaging.incoming.offering-sync-canonical-umb.address=${OFFERING_UMB_CANONICAL_QUEUE}
+mp.messaging.incoming.offering-sync-canonical-umb.client-options-name=umb
+mp.messaging.incoming.offering-sync-canonical-umb.enabled=${UMB_ENABLED}
+mp.messaging.incoming.offering-sync-canonical-umb.failure-strategy=accept
+
+%dev.mp.messaging.connector.smallrye-amqp.client-options.umb.host=localhost
+%dev.mp.messaging.connector.smallrye-amqp.client-options.umb.port=5672
+%dev.mp.messaging.connector.smallrye-amqp.client-options.umb.ssl=false
+%dev.mp.messaging.connector.smallrye-amqp.client-options.umb.saslEnabled=false
+%dev.mp.messaging.connector.smallrye-amqp.client-options.umb.username=guest
+%dev.mp.messaging.connector.smallrye-amqp.client-options.umb.password=guest
```

Comment:
this configuration is necessary to make it work with the artemis service from the docker-compose file? If so, what is default configuration that the smallrye-amqp uses? Asking just in case we can configure the artemis service with it, so we don't need to add these properties here.

Reply:
ssl and sasl had the correct defaults.
With anonymous logins enabled, I was able to delete the username & password.